src/llmcompressor/modifiers/quantization/quantization/base.py

```diff
@@ -292,27 +284,19 @@ def register_calibration_hooks(self, module: Module):
 
         # Calibrate inputs if an input_quant is provided and not running dynamic quant
         if calibrate_inputs:
-            pre_hook_handle = module.register_forward_pre_hook(calibrate_input_hook())
+            self.register_hook(module, calibrate_input_hook, "forward_pre")
 
         if output_quant:
             # hooks for attn modules if running kv_cache quant
             if is_attention_module_:
-                pre_hook_handle = module.register_forward_pre_hook(
-                    calibrate_kv_cache_input_hook(), with_kwargs=True
-                )
-                post_hook_handle = module.register_forward_hook(
-                    calibrate_kv_cache_output_hook()
-                )
+                pre_hook = calibrate_kv_cache_input_hook
+                self.register_hook(module, pre_hook, "forward_pre", with_kwargs=True)
```

Comment:
Do we need the `pre_hook` variable - we can pass in `calibrate_kv_cache_input_hook` directly?

Reply:
That's good as well